test/20_royalties_tests.ts

```diff
@@ -540,4 +550,216 @@ describe('HolographRoyalties Contract', async function () {
 
     it('should allow inherited contract to call fn');
   });
+
+  describe('Royalties Distribution Validation', () => {
+    const errorMargin = ethers.utils.parseUnits('2000', 'wei');
+
+    describe('A collection with 1 recipient', async () => {
+      const totalRoyalties = ethers.utils.parseEther('1');
+
+      before(async () => {
+        const data = (await royalties.populateTransaction.configurePayouts([owner.address], [10000])).data || '';
+        const tx = await l1.factory.connect(owner).adminCall(royalties.address, data);
+        await tx.wait();
+      });
+
+      it('should be able to withdraw all native token balance', async () => {
+        await owner.sendTransaction({
```

Comment:
@alexanderattar I think this should not come from the owner but from somewhere else.

Reply:
where did you have in mind? I think the idea here is it's just funding the contract so there's royalties to pull out. owner is actually `owner = l1.deployer;`. we could change it to be from another account though if it makes more sense